### DISCUSSION TITLE
能搞个其他系统的版本吗？

### DISCUSSION THREAD

**Discussion by 2112360494** [leiurayer/downkyi#67] [Ideas]
主力机太耗电，我有个OMV系统的软路由，但是自带的虚拟机插件好像不支持windous，所以能不能出个Liunx版的或者Docker版的吗？

有偿也行但是本人初二，家境贫寒。。。。。

**Comment by leiurayer** [comment]
跨平台用c#就不合适了，界面框架wpf只能用在windows上，可以换用JAVA或kotlin，或者做成web服务，那就相当于重写了

**Comment by ipiyi** [comment]
> 

你为啥不试试用wine来运行
